lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1255,7 +1282,15 @@ def _deploy(
             deployment_operation = deployments.begin_create_or_update(
                 **deployment_parameters
             )
-            wait_operation(deployment_operation, failure_identity="deploy")
+            while True:
+                try:
+                    wait_operation(
+                        deployment_operation, time_out=480, failure_identity="deploy"
```

Comment:
Why not use 300 seconds, it's easier to understand...

Reply:
updated